fda/grid.py

```diff
@@ -677,92 +685,157 @@ def concatenate(self, other):
                                                         other.data_matrix),
                                                        axis=0))
 
-    def _set_labels(self, ax):
+    def set_figure_and_axes(self):
```

Comment:
Why is this public?

Reply:
It is also used in boxplot and MS-plot.

Reply:
Then maybe it should not be a method, but a function in the plotting module.